docs/advanced/document-store/benchmark.md

```diff
@@ -10,9 +10,9 @@
 :scale: 0 %
 ```
 
-We create a DocumentArray with one million Documents, and benchmark all supported document stores. This includes classic database and vector database all under the same DocumentArray API:
+We create a DocumentArray with one million Documents and benchmark all supported document stores. This includes classic database and vector database, all under the same DocumentArray API:
 
-* {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store". In the sequel, in-memory/memory store refers to this.
+* {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store"
```

Comment:
I guess here the intention is to let users know that we refer to the default usage of `DocumentArray` when we say `in-memory store`

Reply:
Yes, it's said in the first part of the sentence

> * {ref}`"None"<documentarray>`: `DocumentArray()`, namely an in-memory "store"